src/components/Canvas/CanvasTemplate.vue

```diff
@@ -18,26 +18,32 @@
       :imageFactor="imageFactor"
       :focus="true"
     />
+    <!-- This is the initial central image that changes over the time -->
     <image-element
```

Comment:
The first reason is that this image element represents the first central image. For all other cases, this is the central image two blocks before. Why, because when you select an image in a block, an other image block is set and the old one is removed. The problem is that in this case, the central image from this old image block disappear and you cannot roll back in the past. Another solution should be to have three image block and display only one. But that needs to manage the first case and all others to ensure that only the right one is displayed. For this reason, I implement the case like that.

I'm not sure that is clear, feel free to ask me more details.